pkg/service/pipeline.go

```diff
@@ -322,6 +322,34 @@ func (s *service) UpdateNamespacePipelineByID(ctx context.Context, ns resource.N
 		return nil, err
 	}
 
+	toUpdTags := toUpdPipeline.GetTags()
+
+	// TODO: we will need to revert it after the frontend is ready.
```

Comment:
Please be specific in comments like this. Somebody else reading the comment does not necessarily know what "ready" means but would understand "when this feature x is implemented in the fronted". If you are planning to remove this by your self you can use "TODO chuang8511: remove this when the fronted supports updating tags of pipelines. 


 If its a task you are going to solve use "TODO

Reply:
I take it out because it will be merged after the console is released.
- [ref](https://github.com/instill-ai/pipeline-backend/pull/497#issuecomment-2142506640)